api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1938,7 +1940,8 @@ public void makeBucket(String bucketName, String region)
       configString = config.toString();
     }
 
-    executePut(bucketName, null, headerMap, null, US_EAST_1, configString, 0);
+    execute(Method.PUT, US_EAST_1, bucketName, null, headerMap, null,
+            "application/octet-stream", configString, 0);
```

Comment:
We could use `execute()` directly, but these wrappers prevent errors at compile time. Currently response body is not used in any method using `executePut()`.  Closing at `executePut()` prevents file stream leaks else each method should properly `close()`